cmd/vaultworker/res/configuration.toml

```diff
@@ -37,4 +37,4 @@ tokenname4admin = "admin"
 policypath4kong = "res/vault-policy-kong.hcl"
 policyname4kong = "kong"
 tokenname4kong = "kong"
-snis = "edgex.com"
\ No newline at end of file
+snis = "www.edgexfoundry.org"
```

Comment:
missing newline here, but it's okay to fix this later

Reply:
fixed.